IndexNode.h

```diff
@@ -11,7 +11,7 @@ class IndexNode
 	public:
 		const static int INDEX_NODE_SIZE = 64 ;
 		const static int MAX_DIRECT_BLOCKS = 10 ;
-		const static int MAX_FILE_BLOCKS = MAX_DIRECT_BLOCKS ;
+		const static int MAX_FILE_BLOCKS =  MAX_DIRECT_BLOCKS;// + blockSize / 4
```

Comment:
I hardcoded this when I tested, but MAX_FILE_BLOCKS should be proportional to the blockSize. However, I'm not exactly sure how to access the blockSize from the SuperBlock at this point. @elliott-beach Any ideas?

Reply:
I would just remove that constant and call `fileSystem.getBlockSize` in fileDescriptor.

Reply:
Also, just so were on the same page, a test script I would use is 
```
./mkfs filesys.dat 64 64
python -c 'print("x" * 15 * 64)' | ./tee /bigfile
```